samples/identify-graphics/README.metadata.json

```
@@ -0,0 +1,27 @@
+{
+    "category": "Search and Query",
+    "description": "Display a message when a graphic on the map is tapped.",
+    "formal_name": "IdentifyGraphics",
+    "ignore": false,
+    "images": [
+        "identify-graphics.png"
+    ],
+    "keywords": [
+        "geoview-compose",
+        "identify",
+        "graphic"
+    ],
+    "language": "kotlin",
+    "redirect_from": "",
+    "relevant_apis": [
+        "IdentifyGraphicsOverlayResult",
+        "IdentifyGraphicsOverlayResult.graphics",
+        "GraphicsOverlay"
+    ],
+    "snippets": [
+        "src/main/java/com/esri/arcgismaps/sample/identifygraphics/components/MapViewModel.kt",
+        "src/main/java/com/esri/arcgismaps/sample/identifygraphics/screens/MainScreen.kt",
+        "src/main/java/com/esri/arcgismaps/sample/identifygraphics/MainActivity.kt"
```

I would recommend to rename the files and instead to use this pattern of viewmodel/screen for a sample app which should be auto-created when using the [NewModuleScript](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/tree/main/tools#new-module-script): 

 
```suggestion
        "src/main/java/com/esri/arcgismaps/sample/identifygraphics/components/IdentifyGraphicsMapViewModel.kt",
        "src/main/java/com/esri/arcgismaps/sample/identifygraphics/MainActivity.kt",
        "src/main/java/com/esri/arcgismaps/sample/identifygraphics/screens/IdentifyGraphicsMainScreen.kt"
```